src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/block/inscriptiontable/InscriptionTableBlock.java

```diff
@@ -43,6 +46,7 @@ public class InscriptionTableBlock extends Block implements EntityBlock {
     private static final VoxelShape LEFT_Z = AMUtil.joinShapes(box(0, 14, 0, 16, 16, 16), box(0, 13, 0, 15, 14, 1), box(0, 13, 15, 15, 14, 16), box(15, 12, 0, 16, 14, 16), box(12, 5, 6, 13, 11, 10), box(12, 3, 4, 13, 5, 12), box(12, 11, 5, 14, 14, 11), box(11, 0, 0, 13, 5, 4), box(11, 0, 12, 13, 5, 16), box(12, 3, 4, 13, 5, 12));
     private static final VoxelShape RIGHT_X = AMUtil.joinShapes(box(0, 14, 0, 16, 16, 16), box(0, 13, 0, 1, 14, 15), box(15, 13, 0, 16, 14, 15), box(0, 12, 15, 16, 14, 16), box(6, 5, 12, 10, 11, 13), box(4, 3, 12, 12, 5, 13), box(5, 11, 12, 11, 14, 14), box(0, 0, 11, 4, 5, 13), box(12, 0, 11, 16, 5, 13), box(4, 3, 12, 12, 5, 13));
     private static final VoxelShape RIGHT_Z = AMUtil.joinShapes(box(0, 14, 0, 16, 16, 16), box(1, 13, 0, 16, 14, 1), box(1, 13, 15, 16, 14, 16), box(0, 12, 0, 1, 14, 16), box(3, 5, 6, 4, 11, 10), box(3, 3, 4, 4, 5, 12), box(2, 11, 5, 4, 14, 11), box(3, 0, 0, 5, 5, 4), box(3, 0, 12, 5, 5, 16), box(3, 3, 4, 4, 5, 12));
+    private static final String TIER_KEY = "tier";
```

Comment:
maybe use this in the property creation above

Reply:
Will do so.